api/api/tests.py

```diff
@@ -1,5 +1,55 @@
+import functools
+import os
+import pytest
+import sys
+from django.conf import settings
+from django.urls.base import clear_url_caches
+from importlib import import_module, reload
 from rest_framework import status
 from rest_framework.test import APITestCase
+from unittest import mock
+
+
+def reload_root_urls(wrapped):
+
+    @functools.wraps(wrapped)
+    def wrapper(*args, **kwargs):
+        clear_url_caches()
+        if settings.ROOT_URLCONF in sys.modules:
+            reload(sys.modules['api.urls'])
+        else:
+            import_module(settings.ROOT_URLCONF)
+
+        return wrapped(*args, **kwargs)
+
+    return wrapper
+
+
+class AdminTest(APITestCase):
+    """By default, pytest doesn't load .env file"""
```

Comment:
1. Change the name of the test to `AdminEndpointTest`
2. This comment is unrelated to the the test class. The documentation should contain what this particular class is about (what it tests exactly)